configuration/concepts/program_attribute_types.csv

```diff
@@ -22,16 +22,19 @@ bef52504-d77b-485c-ab81-9ccb6eb8f239,,,Pediatrics,Pediatrics,,,,,Misc,N/A,,,,,,,
 589b55a5-40e9-4b7c-8af6-1415b2c6a0bf,,,Needs Surgical Optimization,Needs Surgical Optimization,,,,,Misc,N/A,,,,,,,,,,,
+ef83d1d9-d355-4356-9b60-61217f9dc730,,,Cleared for Surgery,Cleared for Surgery,,,,,Misc,N/A,,,,,,,,,,,
 a20c6c3b-d504-43e8-9ced-bbf788a1f9a8,,,Goldman Score,Goldman Score,,,,,Misc,Coded,,One;Two;Three;Four,,,,,,,,,
 5c5d8fe4-27d8-4a40-832d-506cdc337954,,,Ongoing Care,Ongoing Care,,,,,Misc,N/A,,,,,,,,,,,
 651c5bd1-debd-4485-9bc8-33dd8394775a,,,Surgical Optimization,Surgical Optimization,,,,,Misc,N/A,,,,,,,,,,,
 2873ed4a-c68c-43a8-aede-7e8f6f6dd602,,,Regular Followup,Regular Followup,,,,,Misc,N/A,,,,,,,,,,,
 b392a997-15b7-44b4-819b-8f8db23047aa,,,Consults Service,Consults Service,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental,,,,,,,,,
 5f16a893-d300-4634-9c45-9485be023f98,,,Primary Care Service,Primary Care Service,,,,,Misc,Coded,,Family Medicine;Internal Medicine;Pediatrics;Cardiology;Dermatology;Gynecology;Dental;Ophthalmology;Wound;Other,,,,,,,,,
-d47ce963-77fa-494a-ab37-7f0806d41ecb,,,Priority 1,Priority 1,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental,,,,,,,,,
-c69ff88e-309b-41a0-99ae-5eb58dfc5966,,,Priority 2,Priority 2,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental,,,,,,,,,
-8a6d5b95-ac55-41c2-b27e-ce18012bf962,,,Priority 3,Priority 3,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental,,,,,,,,,
+0c401acb-5188-4bb5-8f6b-e5fbc28a6abf,,,Procedure,Procedure,,,,,Misc,Text,,,,,,,,,,,
+d47ce963-77fa-494a-ab37-7f0806d41ecb,,,Priority 1,Priority 1,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental;Procedure,,,,,,,,,
+c69ff88e-309b-41a0-99ae-5eb58dfc5966,,,Priority 2,Priority 2,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental;Procedure,,,,,,,,,
+8a6d5b95-ac55-41c2-b27e-ce18012bf962,,,Priority 3,Priority 3,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental;Procedure,,,,,,,,,
```

Comment:
'Procedure' is also an appointment_service_type and not an answer to the Priority 1,Priority 2,Priority 3...

Reply:
Done